prow/tide/README.md

```diff
@@ -0,0 +1,131 @@
+# Tide
+
+Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
+component for managing a pool of PRs that match a given set of criteria.
+Automatically retesting PRs in the pool and merging them if they pass tests
+against the latest base branch commit.
+
+Configuration of Tide is located under
+[prow/config.yaml](https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml)
+file. It consists of two parts:
+
+  * **Specification of PR pools/criteria**:
+    A set of all PRs Tide keeps track of.
+    The PRs are periodically checked if they are ready to be merged.
+    Configured under `tide` key of the configuration file.
+
+  * **Specification of PR tests**:
+    A set of jobs that are run over each PR before it gets merged.
+    No job is required to be provided in which case it's enough a PR has all
+    relevant labels set. If a set of jobs is specified,
+    all must succeed (unless configured otherwise).
+    Configured under `presubmits` key of the configuration file.
+
+## PR pools
+
+The set of criteria is specified through the following collection of items:
+
+```yaml
+tide:
+  queries:                // List of queries
+  - repo:                   // org/repo
+    labels:                 // List of must have labels
+    missingLabels:          // List of can't have labels
+    excludedBranches:       // Ignore branches
+    includedBranches:       // Include-only branches
+    reviewApprovedRequired: // Review approved is must
+  sync_period:            // Sync jobs period
+  status_update_period:   // PR status update period
+  merge_method:           // Squash, rebase or just merge PR
+  target_url:             // URL linked to from the details link on the Github status context
+  pr_status_base_url:     // PR status page
+  max_goroutines:         // Per-pool parallelism
+```
+
+Depending on your criteria, some of the items may by omitted.
+
+See [prow/config/tide.go](https://github.com/kubernetes/test-infra/blob/master/prow/config/tide.go)
```

Comment:
I know that @fejta asked for this but isn't it more friendly to point to godoc instead of the source code?

https://godoc.org/github.com/kubernetes/test-infra/prow/config#Tide

Reply:
This is what I was looking for. Thanks!